modules/apps/frontend-js/frontend-js-a11y-web/src/main/java/com/liferay/frontend/js/a11y/web/internal/configuration/A11yConfiguration.java

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.frontend.js.a11y.web.internal.configuration;
+
+import aQute.bnd.annotation.metatype.Meta;
+
+import com.liferay.portal.configuration.metatype.annotations.ExtendedObjectClassDefinition;
+
+/**
+ * @author Matuzalem Teles
+ */
+@ExtendedObjectClassDefinition(category = "infrastructure")
+@Meta.OCD(
+	description = "a11y-configuration-description",
+	id = "com.liferay.frontend.js.a11y.web.internal.configuration.A11yConfiguration",
+	localization = "content/Language", name = "a11y-configuration-name"
+)
+public @interface A11yConfiguration {
+
+	@Meta.AD(
+		deflt = "body", description = "target-description",
+		name = "target-name", required = false
+	)
+	public String target();
+
+	@Meta.AD(deflt = "false", name = "control-menu-name", required = false)
+	public boolean enableControlMenu();
+
+	@Meta.AD(deflt = "false", name = "global-menu-name", required = false)
+	public boolean enableGlobalMenu();
+
+	@Meta.AD(deflt = "false", name = "product-menu-name", required = false)
+	public boolean enableProductMenu();
+
+	@Meta.AD(deflt = "false", name = "editors-name", required = false)
+	public boolean enableEditors();
+
+	@Meta.AD(
+		description = "denylist-description", name = "denylist-name",
+		required = false
+	)
+	public String[] denylist();
+
+	@Meta.AD(
+		description = "axe-core-run-only-description",
+		name = "axe-core-run-only-name", required = false
+	)
+	public String[] axeCoreRunOnly();
+
+	@Meta.AD(
+		deflt = "violations|inapplicable|passes|incomplete",
+		description = "axe-core-result-types-description",
+		name = "axe-core-result-types-name", required = false
+	)
+	public String[] axeCoreResultTypes();
```

Comment:
I have added this setting but I think the most ideal setting for this setting in the UI should be a multi select or checkbox list, not sure which type to use here for that.

Reply:
I may have missed it, however, I glanced over tool code, and I _think_ we're reporting just the violations. So, I don't see how asking axe to give other results would be useful here. We may then omit this config item until we do something with the other result types. That may happen in future iterations if we prioritize report completeness

Reply:
Hmm, I think it's a good idea, but I have no idea how to implement this 😅, well, I can do some research and see what I can do

Reply:
hmm yeah, I think I'm going to remove that from the config and just keep violations.